tests/hypothesis_strategies/kernels.py

```diff
@@ -27,8 +51,66 @@ class KernelType(Enum):
 )
 """A strategy that generates Matern kernels."""
 
+periodic_kernels = st.builds(
+    PeriodicKernel,
+    lengthscale_prior=st.one_of(st.none(), priors),
+    lengthscale_initial_value=st.one_of(st.none(), finite_floats()),
+    period_length_prior=st.one_of(st.none(), priors),
+    period_length_initial_value=st.one_of(st.none(), finite_floats()),
+)
+"""A strategy that generates periodic kernels."""
+
+piecewise_polynomial_kernels = st.builds(
+    PiecewisePolynomialKernel,
+    q=st.integers(min_value=0, max_value=3),
```

Comment:
Not sure if we prefer this, but could actually be:
```suggestion
    q=st.one_of(st.none(), st.integers(min_value=0, max_value=3)),
```
similar to the other args

Reply:
kept as discussed